upstream commit 747b88a1f3cf17150ffba57fe547f3b2cbdefbcb bumped the CMake
requirements to 3.21, but we're still running 3.19, and this was done for
the RUNTIME_DEPENDENCY_SET feature, which is just used on Windows anyway

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@
   limitations under the License.
 ]=========================================================================]
 
-cmake_minimum_required(VERSION 3.21)
+cmake_minimum_required(VERSION 3.19)
 
 project( sonivox
     LANGUAGES C CXX
@@ -38,7 +38,6 @@
 option(ZLIB_SUPPORT "Enable XMF ZLIB Unpacker support" TRUE)
 option(EAS_WT_SYNTH "Enable WaveTable Synth" TRUE)
 option(EAS_FM_SYNTH "Enable FM Synth" TRUE)
-option(INSTALL_DEPENDENCIES "Deploy dependency libraries" FALSE)
 
 if (NOT (EAS_WT_SYNTH OR EAS_FM_SYNTH OR EAS_HYBRID_SYNTH))
     message(FATAL_ERROR "At least one synthesizer type must be enabled: EAS_WT_SYNTH, EAS_FM_SYNTH, EAS_HYBRID_SYNTH.")
@@ -388,7 +387,6 @@
 
 install( TARGETS ${SONIVOX_TARGETS}
     EXPORT sonivox-targets
-    RUNTIME_DEPENDENCY_SET sonivox-dependencies
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
             COMPONENT sonivox_runtime
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
@@ -496,16 +494,6 @@
 if (BUILD_APPLICATION)
     set(sonivox_DIR ${CMAKE_CURRENT_BINARY_DIR})
     add_subdirectory(example)
-endif()
-
-if (INSTALL_DEPENDENCIES)
-    message(STATUS "DEPENDENCY_DIRS: ${DEPENDENCY_DIRS}")
-    install(RUNTIME_DEPENDENCY_SET sonivox-dependencies
-        COMPONENT sonivox_runtime
-        PRE_INCLUDE_REGEXES $<$<BOOL:${_ZLIB_UNPACKER}>:.*zlib.*>
-        PRE_EXCLUDE_REGEXES .*
-        DIRECTORIES ${CMAKE_CURRENT_BINARY_DIR} ${DEPENDENCY_DIRS}
-    )
 endif()
 
 # CPack support
--- a/example/CMakeLists.txt
+++ b/example/CMakeLists.txt
@@ -26,7 +26,6 @@
 target_link_libraries ( sonivoxrender sonivox::sonivox )
 
 install( TARGETS sonivoxrender
-    RUNTIME_DEPENDENCY_SET sonivox-dependencies
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     COMPONENT sonivox_application
 )
